packages/e2e/env/.env.wallet

```diff
@@ -15,7 +15,3 @@ TX_SUBMIT_PROVIDER=blockfrost
 UTXO_PROVIDER=blockfrost
 WALLET_PROVIDER=blockfrost
 STAKE_POOL_PROVIDER=stub
```

Comment:
Is it still `stub`? Or is `STAKE_POOL_PROVIDER` no longer in use?

Reply:
There's still a stub, but it will be removed as part of @AngelCastilloB's work. He'll update this example within ADP-2055, which is his current WIP. He has confirmed this refactor is working on his local branch using the local network too BTW

Reply:
If dependent work is WIP, I think below stub related parameters should be remained as it will make the tests are failed when running against `testnet`.

Reply:
Still `stub` when using Blockfrost, but this will be parked soon, so testing here is limited to the `cardano-services` providers. @AngelCastilloB will be updating the examples in his branch.